tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -54,22 +53,41 @@ COPY --from={{ app["name"] }}-src /openedx/app /openedx/app
 COPY --from={{ app["name"] }}-i18n /openedx/app/src/i18n/messages /openedx/app/src/i18n/messages
 ENV PUBLIC_PATH='/{{ app["name"] }}/'
 EXPOSE {{ app['port'] }}
-CMD ["npm", "run", "start", "---", "--config", "./webpack.dev-tutor.config.js"]
-
-{% endfor %}
-
-{% for app in iter_values_named(suffix="MFE_APP") %}
 
-######## {{ app["name"] }} (production)
-FROM {{ app["name"] }}-dev AS {{ app["name"] }}
 COPY ./env/production /openedx/env/production
-RUN touch /openedx/env/production.override \
+{%- set overrides = app.get("env", {}).get("production", {}) %}
+{%- if overrides %}
+RUN echo "setting production overrides..." \
   {%- for key, value in app.get("env", {}).get("production", {}).items() %}
-  && echo "{{ key }}='{{ value }}'" >> /openedx/env/production.override \
+  && echo "{{ key }}='{{ value }}'" >> /openedx/env/production \
   {%- endfor %}
   && echo "done setting production overrides"
-RUN bash -c "set -a && source /openedx/env/production && source /openedx/env/production.override && npm run build"
+{%- endif %}
+
+######## {{ app["name"] }} (dev)
+FROM {{ app["name"] }}-common AS {{ app["name"] }}-dev
+COPY ./env/development /openedx/env/development
+{%- set overrides = app.get("env", {}).get("development", {}) %}
+{%- if overrides %}
+RUN echo "setting development overrides..." \
+  {%- for key, value in overrides.items() %}
+  && echo "{{ key }}='{{ value }}'" >> /openedx/env/development \
+  {%- endfor %}
+  && echo "done setting development overrides"
+{%- endif %}
+CMD ["/bin/bash", "-c", "set -a && \
+  source /openedx/env/production && \
+  source /openedx/env/development && \
+  npm run start --- --config ./webpack.dev-tutor.config.js"]
```

Comment:
:thinking: this change was introduced a couple days ago by @ARMBouhali: https://github.com/overhangio/tutor-mfe/pull/74/files#r1015250142 In my understanding, it's necessary because it's the only way to use the webpack dev config.

Is there an alternative solution?

Reply:
At the very least, @ARMBouhali, can you point us to how/where you figured this one out?